src/concepts/glean/glean.md

```diff
@@ -77,7 +77,6 @@ Because Glean knows more about the individual data, such as its type and the ran
 
 * _(Work in progress)_ Use events and [Amplitude](https://sso.mozilla.com/amplitude) for product analytics.
 * [Use Databricks](https://sso.mozilla.com/databricks) for deep-dive analysis.
-* [Use the Glean debug ping viewer](debug_ping_view.md) for QA & development.
```

Comment:
I was a little on the fence about whether to keep this link, but decided to take it out. I think it's better just to refer people to the Glean documentation for this type of developer/QA facing stuff, as it's a pain to keep these links up to date.

Reply:
Yeah -- I think that's the right approach.